tests/rptest/services/redpanda.py

```diff
@@ -109,9 +118,40 @@ def start_node(self, node, override_cfg_params=None):
                 timeout_sec=RedpandaService.READY_TIMEOUT_SEC,
                 backoff_sec=0.5,
                 err_msg=
-                "Redpanda didn't finish startup in {RedpandaService.READY_TIMEOUT_SEC} seconds",
+                f"Redpanda didn't finish startup in {RedpandaService.READY_TIMEOUT_SEC} seconds",
+            )
+
+    def coproc_enabled(self):
+        coproc = self._extra_rp_conf.get('enable_coproc')
+        dev_mode = self._extra_rp_conf.get('developer_mode')
+        return coproc is True and dev_mode is True
+
+    def start_wasm_engine(self, node):
+        wcmd = (f"nohup {self.find_wasm_root()}/bin/node"
+                f" {self.find_wasm_root()}/output/modules/rpc/service.js"
+                f" {RedpandaService.CONFIG_FILE} "
+                f" >> {RedpandaService.WASM_STDOUT_STDERR_CAPTURE} 2>&1 &")
+
+        self.logger.info(
+            f"Starting wasm engine on {node.account} with command: {wcmd}")
+
+        # wait until the wasm engine has finished booting up
+        with node.account.monitor_log(
+                RedpandaService.WASM_STDOUT_STDERR_CAPTURE) as mon:
+            node.account.ssh(wcmd)
+            mon.wait_until(
+                "Starting redpanda wasm service on port: 43189",
```

Comment:
maybe remove the port from the search string (or set it in a formatted string based on config)